src/pages/_app.tsx

```diff
@@ -44,6 +44,16 @@ export default function App({ Component, pageProps }: AppProps) {
         />
         <meta name="twitter:site" content="@nypl" />
         <meta name="twitter:creator" content="@nypl" />
+        <meta
+          name="google-site-verification"
+          content="_shbOK1otHA_eFNFgJsOwITZrWQwRg4wr8nmrJPNVDM"
```

Comment:
I _think_ this is okay. I don't know enough about how Google verifies sites and if you can reuse the same hash on multiple sites that have different domain names. At some point, this app will go on "digitalcollections.nypl.org" so by then it won't be a problem and it's likely fine to keep this here now.